api/Controllers/AccountController.cs

```diff
@@ -0,0 +1,99 @@
+using api.Interfaces;
+using api.Models;
+using api.Dto.Account;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+
+namespace api.Controllers
+{
+    [Route("api/account")]
+    [ApiController]
+    public class AccountController : ControllerBase
+    {
+        private readonly UserManager<AppUser> _userManager;
+        private readonly ITokenService _tokenService;
+        private readonly SignInManager<AppUser> _signManager;
+
+        public AccountController(UserManager<AppUser> userManager, ITokenService tokenService, SignInManager<AppUser> signInManager)
+        {
+            _userManager = userManager;
+            _tokenService = tokenService;
+            _signManager = signInManager;
+        }
+
+        [HttpPost("login")]
+        public async Task<IActionResult> Login(LoginDto loginDto)
+        {
+            if(!ModelState.IsValid)
+                return BadRequest(ModelState);
+
```

Comment:
```suggestion
            if(!ModelState.IsValid) {
                return BadRequest(ModelState);
            }
            
```

Так более понятно будет)